### PR TITLE
BF-021 task finished, (Watchlist route)

### PR DESCRIPTION
**Pull request by Migueld1324**
BF-021 task finished, (Watchlist route), modified app.module.js adding the route, changing the route on topbar.directive.js to link it to the new route, and created the template for our new route.

### CODE REVIEW

File: blimflix/client/app/core/sidebar.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@
   top: 50px;
```
Comment (lordpixel) [inline blimflix/client/app/core/sidebar.scss:6]: @Migueld1324 please do not include unrelated code into this PR, it should only contain code from watchlist.

Comment (Migueld1324) [reply blimflix/client/app/core/sidebar.scss:6]: I think when I did the rebase from develop those files merged into my branch and when I did the push the files were uploaded to the PR, but I didn't modified them.
